src/js/components/header/sfLogin.tsx

```diff
@@ -29,39 +28,52 @@ interface MenuOption {
 const isDivider = (opt: MenuOption | Divider): opt is Divider =>
   (opt as Divider).type === 'divider';
 
-class Login extends React.Component<Props, { modalOpen: boolean }> {
-  public static defaultProps = {
-    id: 'login',
-    buttonClassName: 'slds-button_outline-brand',
-    buttonVariant: 'base',
-    disabled: false,
-    menuPosition: 'overflowBoundaryElement',
-    nubbinPosition: 'top right',
-  };
+const getMenuOpts = (): (MenuOption | Divider)[] => [
```

Comment:
not sure where  or if I can see these options?

Reply:
Nope, you can't. :)

This file is hanging around but unused for the moment -- until we add SF login. (It's the same login dropdown as MD, functionally.)